### PR TITLE
Done with heroSection for Homepage and added homepage assets

### PR DESCRIPTION
**Pull request by nathnaeltefera**
Screenshot 2023-12-22 165539

### CODE REVIEW

**Review by Joephy527 — COMMENTED** [review]
poppins font is already defined in layout so no need to do it again in the page file

importing all the image files at the top is a bit messy, better to just do it at the source prop of the image component

use prettier to format your code, it'll get rid of the inconsistencies like using double quotes sometimes and single quotes other times

clean up the unnecessary whitespaces like the ones at the end and start of classNames

#707070 is already configured as Quinary in the colors property in the tailwind config file so use that instead of putting it as a custom color property.

add a few comments through out for clarity

and also provide images of responsiveness

File: app/components/homePage/GetStarted.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import Link from "next/link"
+import ContainerVector from '@/public/assets/images/HomePage/Vector.svg'
+import dynamic from 'next/dynamic'
+
+const LottiePlayer = dynamic(() => import('./LottiePlayer'), {
+    ssr: false,
+})
+
+
```
Comment (BrookW-83) [inline app/components/homePage/GetStarted.tsx]: Use prettier format remove this unessary space in all fiels

File: app/green.json
Comment (BrookW-83) [inline app/green.json:1]: What is this file, remove it

File: public/assets/animations/homePage/dzdrsyvLv7.json
Comment (BrookW-83) [inline public/assets/animations/homePage/dzdrsyvLv7.json:1]: Use snake-case for images folder

File: app/page.tsx
Comment (BrookW-83) [inline app/page.tsx:1]: You need to use your own folder for the page file, Creat a pages folder name it your pages name, then inside it put the pages file

Comment (BrookW-83) [reply app/page.tsx:1]: Homepage/pages.tsx

File: app/components/homePage/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                            src={triangularPrism}
+                            alt='triangularPrism'
+                        />
+                        <div className=' [font-size:_clamp(25px,3vw,100px)] text-[#707070]'>Welcome to ESSS</div>
+                        <Image
+                            className='[size:_clamp(25px,3vw,100px)]'
+                            src={learningLogo}
+                            alt='learning logo'
+                        />
+                        <button className=" text-Secondary  lg:mr-11 mt-6   bg-Primary rounded-full px-2.5 py-2 md:px-4 md:py-2  lg:px-5 lg:py-3 hover:bg-Secondary hover:text-Primary lg:text-2xl md:text-xl " >
```
Comment (BrookW-83) [inline app/components/homePage/HeroSection.tsx]: Remove unsessary sapce between tailwind code

File: app/components/homePage/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                </div>
+                <div className="  flex items-end justify-center w-full flex-1 ">
+                    <div className='relative bottom-0 max-w-[600px] right-0 w-full aspect-[6/7]  '>
+                        <Image
+                            className='absolute bottom-0 '
+                            // fill
+                            src={femaleTeacher}
+                            alt='femaleTeacher'
+                        />
+
```
Comment (BrookW-83) [inline app/components/homePage/HeroSection.tsx]: Remove unnessary white space

File: app/components/homePage/GetStarted.tsx
Comment (BrookW-83) [inline app/components/homePage/GetStarted.tsx:1]: Use prettier format to clean up your code ,

File: app/components/homePage/GetStarted.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import Link from "next/link"
+import ContainerVector from '@/public/assets/images/HomePage/Vector.svg'
+import dynamic from 'next/dynamic'
+
+const LottiePlayer = dynamic(() => import('./LottiePlayer'), {
```
Comment (BrookW-83) [inline app/components/homePage/GetStarted.tsx]: If it makes no change can you put this function inside the arrow function for cleaner code

File: app/components/homePage/HeroSection.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+import femaleTeacher from '@/public/assets/images/HomePage/Group 33@2x.png'
+import cube from '@/public/assets/images/HomePage/Group 16.svg'
+import circles from '@/public/assets/images/HomePage/Group 19.svg'
+import hexagon from '@/public/assets/images/HomePage/Group 5.svg'
+import triangularPrism from '@/public/assets/images/HomePage/Group 14.svg'
+import hexagonFilled from '@/public/assets/images/HomePage/Path 707.svg'
+import rotatedTriangle from '@/public/assets/images/HomePage/Polygon 1.svg'
+
+import Link from 'next/link'
+
```
Comment (BrookW-83) [inline app/components/homePage/HeroSection.tsx]: Remove this unnessary space

**Review by BrookW-83 — COMMENTED** [review]
creat a cleaner code,

File: app/page.tsx
Comment (Joephy527) [reply app/page.tsx:1]: the root page file is used as the home destination so no need to create a folder

File: app/components/homePage/Card.tsx
Comment (BrookW-83) [inline app/components/homePage/Card.tsx:1]: change the card name to more discriptive name

File: app/components/homePage/GetStarted.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+import dynamic from "next/dynamic";
+
+const LottiePlayer = dynamic(() => import("./LottiePlayer"), {
+  ssr: false,
+});
+
+const GetStarted = () => {
+  return (
+    <section className="flex flex-col bg-Secondary md:flex-row min-h-[300px] h-[30vh] ">
+      <div className="flex flex-1  h-full justify-center items-center">
```
Comment (BrookW-83) [inline app/components/homePage/GetStarted.tsx]: remove whitespace btween h-full and flex-1

File: app/components/homePage/HeroSection.tsx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+            src={hexagon}
+            alt="hexagon"
+          />
+          <Image
+            className="absolute max-w-[100px] w-1/6 top-0 left-1/2 "
+            src={rotatedTriangle}
+            alt="rotatedTriangle"
+          />
+          <Image
+            className="absolute min max-w-[80px] w-2/12 bottom-1/2 translate-y-[400%]  translate-x-1/2 right-0 "
```
Comment (BrookW-83) [inline app/components/homePage/HeroSection.tsx]: remove white space

File: app/components/homePage/HeroSection.tsx [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+import triangularPrism from "@/public/assets/images/HomePage/Group 14.svg";
+import hexagonFilled from "@/public/assets/images/HomePage/Path 707.svg";
+import rotatedTriangle from "@/public/assets/images/HomePage/Polygon 1.svg";
+
+import Link from "next/link";
+
+const HeroSection = () => {
+  return (
+    <section className="flex h-[40vh] min-h-[450px] md:min-h-[600px] md:h-[70vh] lg:h-[95vh]  bg-Tertiary ">
+      <div className="flex w-full ">
```
Comment (BrookW-83) [inline app/components/homePage/HeroSection.tsx]: remove white space at the end

File: app/components/homePage/HeroSection.tsx [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+import hexagon from "@/public/assets/images/HomePage/Group 5.svg";
+import triangularPrism from "@/public/assets/images/HomePage/Group 14.svg";
+import hexagonFilled from "@/public/assets/images/HomePage/Path 707.svg";
+import rotatedTriangle from "@/public/assets/images/HomePage/Polygon 1.svg";
+
+import Link from "next/link";
+
+const HeroSection = () => {
+  return (
+    <section className="flex h-[40vh] min-h-[450px] md:min-h-[600px] md:h-[70vh] lg:h-[95vh]  bg-Tertiary ">
```
Comment (BrookW-83) [inline app/components/homePage/HeroSection.tsx]: remove white space between lg abd bg
remove white space at the end

File: app/components/homePage/GetStarted.tsx [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+            <LottiePlayer />
+          </div>
+          <button className=" text-Secondary mt-4 px-4 py-2  lg:px-5 lg:py-3 lg:text-2xl text-xl  bg-Primary rounded-full  hover:bg-Secondary hover:text-Primary  ">
+            <Link href="/register">Register Now</Link>
+          </button>
+        </div>
+      </div>
+      <div className="flex flex-1 h-full  ">
+        <div className="flex ml-3 w-full md:w-[80%] justify-center text-center  md:text-left flex-col ">
+          <h1 className="font-AbolitionTestRegular font-bold  [font-size:_clamp(45px,6vw,100px)]  tracking-wider">
```
Comment (BrookW-83) [inline app/components/homePage/GetStarted.tsx]: remove white space between font-bold and [..

File: app/components/homePage/GetStarted.tsx [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+const GetStarted = () => {
+  return (
+    <section className="flex flex-col bg-Secondary md:flex-row min-h-[300px] h-[30vh] ">
+      <div className="flex flex-1  h-full justify-center items-center">
+        <div className=" flex flex-col  items-center  w-2/3">
+          <div className=" w-1/3 md:w-1/2  rotate-90">
+            <LottiePlayer />
+          </div>
+          <button className=" text-Secondary mt-4 px-4 py-2  lg:px-5 lg:py-3 lg:text-2xl text-xl  bg-Primary rounded-full  hover:bg-Secondary hover:text-Primary  ">
```
Comment (BrookW-83) [inline app/components/homePage/GetStarted.tsx]: remove white space in this line

File: app/components/homePage/Card.tsx [hunk 12]
```diff
@@ -0,0 +1,15 @@
+interface CardProps {
```
Comment (BrookW-83) [inline app/components/homePage/Card.tsx]: rename cardprops to more disscriptive name

File: app/components/homePage/GetStarted.tsx (hunk 6 above)
Comment (BrookW-83) [reply app/components/homePage/GetStarted.tsx]: remove white space at the end of the of "

File: app/components/homePage/LottiePlayer.tsx [hunk 13]
```diff
@@ -0,0 +1,15 @@
+'use client'
+import Lottie from "lottie-react";
+import animatedArrow from '@/public/assets/animations/homePage/dzdrsyvLv7.json'
+
+
+const LottiePlayer = () => {
+  return (
+    <>
+        <Lottie animationData={animatedArrow} loop={true} />
+
```
Comment (BrookW-83) [inline app/components/homePage/LottiePlayer.tsx]: remove the empty line

File: public/assets/images/HomePage/Astro101@2x.png
Comment (BrookW-83) [inline public/assets/images/HomePage/Astro101@2x.png:1]: change you image naming to snakecase

**Review by BrookW-83 — COMMENTED** [review]
Remove white spaces
Change the naming scheme to snakecase for all images

File: app/components/home_Page/ExploreSection.tsx [hunk 14]
```diff
@@ -0,0 +1,45 @@
+import TopicCard from "./TopicCard";
+
+const cardItems = [
```
Comment (BrookW-83) [inline app/components/home_Page/ExploreSection.tsx]: - why do you have a list here, put it inside a data folder and import them

File: app/components/home_Page/ExploreSection.tsx [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+      <div className="flex bg-Tertiary lg:w-1/2 items-center justify-center flex-1">
+        <div className="  flex gap-3 h-40 justify-center items-center lg:flex-none lg:flex-col lg:items-start tracking-wider font-AbolitionTestRegular leading-none [font-size:_clamp(45px,6vw,100px)]  w-2/3">
+          <h1 className="uppercase">Find</h1>
+          <h1 className="uppercase not-italic lg:italic ml-[-12px]">What Interests</h1>
+          <h1 className="uppercase text-Secondary tracking-wider">You</h1>
+        </div>
+      </div>
+      <div className=" h-[80%] lg:h-full lg:w-1/2 gap-10 ">
+        <div className="overflow-x-auto h-full grid grid-flow-col md:grid md:grid-rows-2  items-center md:justify-around">
+
```
Comment (BrookW-83) [inline app/components/home_Page/ExploreSection.tsx]: remove this white spaces

File: app/components/home_Page/GetCertified.tsx [hunk 16]
```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+
+const GetCertified = () => {
+  return (
+
```
Comment (BrookW-83) [inline app/components/home_Page/GetCertified.tsx]: -remove this white space

File: app/components/home_Page/LottiePlayer.tsx [hunk 17]
```diff
@@ -0,0 +1,15 @@
+'use client'
+import Lottie from "lottie-react";
+import animatedArrow from '@/public/assets/animations/home_Page/arrow.json'
+
+
+const LottiePlayer = () => {
+  return (
+    <>
+        <Lottie animationData={animatedArrow} loop={true} />
```
Comment (BrookW-83) [inline app/components/home_Page/LottiePlayer.tsx]: -remove white spaces

**Review by BrookW-83 — COMMENTED** [review]
- change the names of the images to snake case

File: app/components/home_Page/dummyData.js
Comment (BrookW-83) [inline app/components/home_Page/dummyData.js:1]: - creat a data folder and then put it there
- rename from dummydata to descriptive name,

Comment (BrookW-83) [reply app/components/home_Page/dummyData.js:1]: and it should be named ts file

File: app/components/home_Page/MentorCard.tsx [hunk 18]
```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+
+interface CardProps {
```
Comment (BrookW-83) [inline app/components/home_Page/MentorCard.tsx:3]: - rename to MentorCardProps

File: app/components/home_Page/dummyData.js
Comment (BrookW-83) [reply app/components/home_Page/dummyData.js:1]: - The data folder should be outside of app folder, above public folder

**Review by BrookW-83 — APPROVED** [review]

**Pull request merged**